extract/rpm-package.go

```diff
@@ -51,14 +50,14 @@ func fixKernelModulesSymlinks(directory, kernelUname string, l types.Logger) {
 			}
 
 			if err := os.Remove(symlink); err != nil {
-				l.Warnf("failed to unlink symlink at %s: %v", symlink, err)
+				l.Warnf("failed to unlink symlink at %s: %w", symlink, err)
```

Comment:
I think `%w` only works with `fmt.Errorf`, so you probably want `%s` or `%v` here.